elastic/kibana
# [Infrastructure UI] Improve alerts query and fix import

## 📓 Summary

Closes #152544 

The issue was caused by a wrong import, where we were directly importing the hook implementation instead of the context hook exposed by `constate`.

This PR also improves a bit the alerts status filtering switching from `match_phrase` to a `term` query.

## 🧪 Testing
- Navigate to the Hosts View
- Open dev tools
- Refresh the page or trigger a new search
- Verify the `/snapshot` API is called once to retrieve the table metrics
